packages/blockchain-extension/extension/commands/importNodesToEnvironmentCommand.ts

```diff
@@ -128,32 +129,52 @@ export async function importNodesToEnvironment(environmentRegistryEntry: FabricE
                 }
             }
         } else {
-            const keytar: any = ModuleUtil.getCoreNodeModule('keytar');
-            if (!keytar) {
-                throw new Error('Error importing the keytar module');
-            }
-
             const GET_ALL_COMPONENTS: string = '/ak/api/v1/components';
             const api: string = environmentRegistryEntry.url.replace(/\/$/, '') + GET_ALL_COMPONENTS;
             let response: any;
+            let requestOptions: any;
 
+            // establish connection to Ops Tools
             try {
-                // establish connection to Ops Tools and retrieve json file
-                const credentials: string = await keytar.getPassword('blockchain-vscode-ext', environmentRegistryEntry.url);
-                const credentialsArray: string[] = credentials.split(':');
-                if (credentialsArray.length !== 3) {
-                    throw new Error(`Unable to retrieve the stored credentials`);
+                if (environmentRegistryEntry.environmentType === EnvironmentType.SAAS_OPS_TOOLS_ENVIRONMENT) {
+                    const cloudAccountExtension: vscode.Extension<any> = vscode.extensions.getExtension( 'IBM.ibmcloud-account' );
+                    if ( !cloudAccountExtension.isActive ) {
```

Comment:
Can we split this code out into a separate file as it seems duplicated, also why is there less log in code here

Reply:
done. Login code is the same, but more work is needed to get the console endpoint.